Gemfile

```diff
@@ -1,2 +1,5 @@
 source 'https://rubygems.org'
 gemspec
+
+gem 'pry'
```

Comment:
Though I like both of these tools, I'd prefer to not them to the _gemspec_ at this time.